azkaban-common/src/main/java/azkaban/imagemgmt/dto/ImageTypeDTO.java

```diff
@@ -65,6 +65,7 @@ public void setDeployable(final Deployable deployable) {
   }
 
   public void setOwnerships(final List<ImageOwnershipDTO> ownerships) {
-    this.ownerships = ownerships;
+            ownerships.stream().forEach(o -> o.setName(getName()));
```

Comment:
Why is this needed?

Reply:
This is needed becuase the imageOwnershipDTO doesn't get its name field populated. It shows as "null" in the request.